firmware/baseband/proc_btlerx.cpp

```diff
@@ -26,245 +27,476 @@
 
 #include "event_m4.hpp"
 
+uint32_t BTLERxProcessor::crc_init_reorder(uint32_t crc_init) {
+    int i;
+    uint32_t crc_init_tmp, crc_init_input, crc_init_input_tmp;
+
+    crc_init_input_tmp = crc_init;
+    crc_init_input = 0;
+
+    crc_init_input = ((crc_init_input) | (crc_init_input_tmp & 0xFF));
```

Comment:
nit: why are you OR'ing with `crc_init_input` here? It's always 0?
`crc_init_input = crc_init & 0xFF;` should be equivalent?

Reply:
If this is swapping the byte-order, add a comment to describe that.
Also that feels like something there should be a library for.

Reply:
You're right changing this.

Reply:
Done